.github/workflows/pull-request-test.yml

```diff
@@ -17,6 +17,17 @@ jobs:
       - name: Build
         run: ./gradlew build
 
+      - uses: actions/checkout@v4
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Build
+        run: ./gradlew build
```

Comment:
This will run the two builds in sequence which is pretty slow. 

Can you please run in parallel instead? The quickest way I think would be to create another workflow that also runs `on: [pull_request]`

Reply:
`strategy.matrix` should run the builds in parallel. Added in https://github.com/amplitude/Amplitude-Kotlin/pull/200/commits/d7f24884a83d603a3f2560cb65fab370fad64bed.